patches/minecraft/net/minecraft/util/Util.java.patch

```diff
@@ -0,0 +1,11 @@
+--- a/net/minecraft/util/Util.java
++++ b/net/minecraft/util/Util.java
+@@ -200,7 +200,7 @@
+       try {
+          type = DataFixesManager.func_210901_a().getSchema(DataFixUtils.makeKey(SharedConstants.func_215069_a().getWorldVersion())).getChoiceType(p_240990_0_, p_240990_1_);
+       } catch (IllegalArgumentException illegalargumentexception) {
+-         field_195650_a.error("No data fixer registered for {}", (Object)p_240990_1_);
++         if (!net.minecraftforge.fml.loading.FMLConfig.disableDatafixerWarning()) field_195650_a.error("No data fixer registered for {}", (Object)p_240990_1_);
```

Comment:
If you put the if statement on its own line above (without changing indentation of the log line the patch will be slightly smaller.

Reply:
Done